robyn/__init__.py

```diff
@@ -56,6 +58,7 @@ def __init__(self, file_object: str, config: Config = Config()) -> None:
         self.event_handlers = {}
         self.exception_handler: Optional[Callable] = None
         self.authentication_handler: Optional[AuthenticationHandler] = None
+        self.rate_limiter = RateLimiter()
```

Comment:
One question - do we always want to have a ratelimiter instance? We can make it optional and handle the cases in the code? Or make as constant argument in the `__init__` function like config

Reply:
I prefer having the RateLimiter instance handle the details of whether to check the limit etc, keep all the responsibilities there, than to handle the cases in the Robyn class.

Reply:
@IdoKendo , alright makes sense. However, can we make it a default argument in the constructor? This will also allow the users to extend the RateLimiter with a custom implementation if they desire to.